src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,34 @@
+import { FC } from 'react';
+import { Person } from '../../types';
+import { PersonInfo } from '../PersonInfo';
+
+interface Props {
+  people: Person[]
+}
+export const PeopleTable: FC<Props> = ({ people }) => {
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th>Sex</th>
+          <th>Born</th>
+          <th>Died</th>
+          <th>Mother</th>
+          <th>Father</th>
```

Comment:
We can cheat a bit here and improve the code if we create an array of column names and return them in a similar way, I'm currently doing with the `people` array and the `PersonInfo` component below.